src/drivers/QtTestDriver.cpp

```diff
@@ -0,0 +1,32 @@
+#include "cucumber-cpp/internal/drivers/QtTestDriver.hpp"
+
+#include <QtTest/QtTest>
+#include <QTextStream>
+#include <QTemporaryFile>
+
+namespace cucumber {
+namespace internal {
+
+const InvokeResult QtTestStep::invokeStepBody() {
+    QTemporaryFile file;
+    QString fileName;
+    if(!file.open()) {
+        return InvokeResult::failure("Unable to open temporary file needed for this test");
+    }
+    file.close();
+
+    QtTestObject testObject(this);
+    int returnValue = QTest::qExec(&testObject, QStringList() << "test" << "-o" << file.fileName());
+    if(returnValue == 0)
```

Comment:
Space between `if` and opening brace please.

Reply:
ok, we need that clang-format PR merged :)

Reply:
You can copy the `.clang-format` file from that PR into your local working copy and execute this to format everything changes since master (replace origin/master with the most recent commit of the master branch that you based your changes on).:
```
git clang-format-3.8 --binary=/usr/bin/clang-format-3.8 --style=file --commit=origin/master
```